suggestions in improving the contributing.md docs

In my earlier attempt in fixing one of the issues, i faced the following challenges which i describe below as a form of updating the contributing.md documentation 


line 25:
  At this point make sure the Developer Mode is enabled in your user settings of Discord
  ```mermaid
  graph LR 
  A(Discord Application) --> B(USER SETTINGS)--> C(APP SETTINGS)--> D(Adanced)--> E(Developer Mode = Enable) 
```
alternatively we could say this at the beginning of the page on line 3  :
`To develop this project locally, you will need to be running [NodeJS v20](https://nodejs.org/en/) , [pnpm v8](https://pnpm.io/). and have enabled Developer Mode in Discord` 

line 40 could be updated to `pnpm i --frozen-lockfile`  or `pnpm install`
`npm ci` refused to work for me showing the following as error
`The npm ci command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.`

line 46 could be updated to `pnpm run build`.
line 73 could be updated to `pnpm start`.
line 83 could be updated to `pnpm run lint`.

**Desktop (please complete the following information):**
 - OS: [MacOS 13.4.1]
 - Browser: [Chrome]
 - Version: [122.0.6261.94 

**Additional context**
I am happy to work on this issue once the correct approach and wording is decided.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -25,7 +25,7 @@ Next, head over to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) and eith
 
 The bot uses two Discord webhooks, one for debug logging and one for moderation action logs. For local testing, you can use the same webhook URL for both. Right-click on a channel in your server, select `Edit Channel`, and pick the `Integrations` tab on the sidebar. Click the `Create Webhook` button, give it a name, and copy the URL. Paste that as `DEBUG_HOOK` and `MOD_HOOK` in your `.env` file.
 
-If you haven't already enabled Developer Mode in your Discord client, You can do this by opening your user/app setting and selecting `Advanced`,  then toggling `Developer Mode` on.
+If you haven't already enabled Developer Mode in your Discord client, you can do this by opening your user/app setting and selecting `Advanced`,  then toggling `Developer Mode` on.
 
 Right-click on your Discord server icon (the one in which you'll be running the bot) and select `Copy ID`. This goes in your `.env` file as `HOME_GUILD`.
 

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -25,6 +25,8 @@ Next, head over to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) and eith
 
 The bot uses two Discord webhooks, one for debug logging and one for moderation action logs. For local testing, you can use the same webhook URL for both. Right-click on a channel in your server, select `Edit Channel`, and pick the `Integrations` tab on the sidebar. Click the `Create Webhook` button, give it a name, and copy the URL. Paste that as `DEBUG_HOOK` and `MOD_HOOK` in your `.env` file.
 
+If you haven't already enabled Developer Mode in your Discord client, You can do this by opening your user/app setting and selecting `Advanced`,  then toggling `Developer Mode` on.
+
 Right-click on your Discord server icon (the one in which you'll be running the bot) and select `Copy ID`. This goes in your `.env` file as `HOME_GUILD`.
 
 > **Note:**
@@ -37,13 +39,13 @@ Finally, for the private conversation system, you need to specify a moderation r
 Install the dependencies:
 
 ```bash
-npm ci
+pnpm i --frozen-lockfile
 ```
 
 Build the code:
 
 ```bash
-npm run build
+pnpm run build
 ```
 
 ## Inviting the Bot
@@ -70,7 +72,7 @@ Discord marks specific gateway intents as "privileged". Back on your application
 Finally, everything should be set up to run the bot. Start it with:
 
 ```bash
-npm start
+pnpm start
 ```
 
 You should see the bot come online, and a message sent to your debug webhook letting you know the bot is ready.
@@ -80,7 +82,7 @@ You should see the bot come online, and a message sent to your debug webhook let
 We are happy to accept contributions to this project. Please ensure that all code contributions follow our standard:
 
 ```bash
-npm run lint
+pnpm run lint
 ```
 
 If you have any issues or questions, feel free to [ask us in Discord](https://discord.gg/KVUmVXA).

